### DISCUSSION TITLE
resolve import assertions error

### DISCUSSION THREAD

**Discussion by TomokiMiyauci** [dotansimha/graphql-code-generator#8621] [Q&A]
The following error occurs when targeting file that uses import assertions.

```bash
This experimental syntax requires enabling the parser plugin: "importAssertions".
````

This seems to be an error from babel, is there any way to resolve this?

**Comment by charlypoly** [comment]
@TomokiMiyauci I moved it to a issue since it is a bug report: https://github.com/dotansimha/graphql-code-generator/issues/8630
